ee/cron/workspaceHostLoads.js

```diff
@@ -169,11 +158,11 @@ async function handleWorkspaceAutoscaling(stats) {
           MaxCount: needed,
           MinCount: 1,
           LaunchTemplate: {
-            LaunchTemplateId: launch_template_id,
+            LaunchTemplateId: config.workspaceLoadLaunchTemplateId,
```

Comment:
Note to self: we'll have to add this to our configs before deploying this change.